### PR TITLE
feat(svg-viewer): add file props to take in raw svg

### PR DESCRIPTION
**Pull request by mrtraser**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-537.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Changed base of #526

**Comment by mrtraser** [comment]
@valgussev [suggestion](https://github.com/cognitedata/gearbox.js/pull/526#pullrequestreview-327573897) from the previous PR seems to be reasonable to implement here than to have few input  props.

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-537.gearboxjs.preview.cogniteapp.com

File: src/components/SVGViewer/SVGViewer.tsx [hunk 1]
```diff
@@ -19,9 +19,11 @@ const wheelZoomLevel = 0.15;
 const currentAssetClassName = 'current-asset';
 const minDesktopWidth = 992;
 
-export interface SvgViewerProps {
+interface SvgViewerProps {
   // CDF fileId to fetch svg-document
   documentId: number;
+  // Function to replace document fetching call
+  fetchDocument?: (documentId: number) => Promise<string>;
```
Comment (polomani) [inline src/components/SVGViewer/SVGViewer.tsx]:
```suggestion
  fetchDocument<T>?: (...params: T) => Promise<string>;
```

Comment (mrtraser) [reply src/components/SVGViewer/SVGViewer.tsx]: `documentId` is something that comes from the inside of component, in this case it is `documentId` prop, which has type `number`

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-537.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Revert to file prop based on @polomani comment – confusing approach when you need to pass `documentId` which is not related to CDF to handle it inside fetch function

**Review by polomani — APPROVED** [review]

File: src/components/SVGViewer/SVGViewer.tsx (hunk 1 above)
Comment (polomani) [reply src/components/SVGViewer/SVGViewer.tsx]: right

**Pull request merged**